dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -84,6 +85,31 @@ static ArgumentParser getArgumentParser() {
         parser.addArgument("-x")
                 .help("comment snippet mentioned in line just before FROM instruction for ignoring a child image. " +
                         "Defaults to 'no-dfiu'");
+        parser.addArgument("-u", "--" + Constants.USE_RATE_LIMITING)
```

Comment:
@afalko  we can combine two of the args(rate limit and rate limit duration) into one say 500s or 500m (500 per min or seconds). But I choose to keep it separated mainly because of 
1. By default this feature is not enabled.
2. Even if it is enabled, the program defaults to a reasonable value.
3. Adding tokenizing the user input and converting it to the required format will have additional maintenance overhead and is prone to bugs over time. So I feel having two arguments will pan out better than combing them into one and tokenizing them ourselves.

Let me know if that sounds reasonable. if not I'll combine two of them into one.

Reply:
I'd rather it be just one rate limit flag exposed to the user. Same feedback I gave in https://github.com/salesforce/dockerfile-image-update/pull/588 . 

Having multiple parameters that work together actually adds additional edge conditions that we have to handle. There is additional cognitive load on our users as well. These are all also top-level flags in the command so the more flags you add the other existing flags are diluted.

Reply:
Just to clarify, I would prefer we combine all 4 into one param.

Reply:
let me work on it today and see what best to merge all of them.